app/models/article.rb

```diff
@@ -112,4 +114,8 @@ def draft!
   def ensure_account_id
     self.account_id = portal&.account_id
   end
+
+  def ensure_article_slug
+    self.slug = title.underscore.parameterize(separator: '-')
```

Comment:
```suggestion
    self.slug = title.underscore.parameterize(separator: '-') if title.present?
```

If the title is empty, it will throw this error instead of the title validation message.

![Screenshot 2022-09-29 at 1 45 26 PM](https://user-images.githubusercontent.com/12408980/192978089-3916f2e9-a171-4084-84c9-c4e223be4ba7.png)